src/db/car/plain.rs

```diff
@@ -431,13 +430,10 @@ mod tests {
         }
     }
 
-    fn reference(reader: impl futures::AsyncRead + Send + Unpin) -> MemoryBlockstore {
+    fn reference(reader: impl AsyncBufRead + Unpin) -> MemoryBlockstore {
```

Comment:
This can be written as:
```rust
   fn reference(reader: impl AsyncBufRead + Unpin) -> MemoryBlockstore {
        let blockstore = MemoryBlockstore::new();
        block_on(load_car(&blockstore, reader)).unwrap();
        blockstore
    }
```